docs/source/toy/index.rst

```diff
@@ -32,6 +32,7 @@ Some toy classes provide extra functionality defined in the
     repressilator_model
     rosenbrock
     simple_egg_box_logpdf
+    simple_harmonic_oscillator_model
```

Comment:
I prefer this to the "sho" model filenames used below. Unless that's a common abbrev.?

Reply:
Yep, common abbreviation!